bezier-rs/lib/src/lib.rs

```diff
@@ -122,57 +135,64 @@ impl Bezier {
 
 	pub fn get_handle1(&self) -> DVec2 {
 		match self.handles {
-			BezierHandles::Quadratic { handle } => {
-				handle
-			}
-			BezierHandles::Cubic { handle1, handle2: _ } => {
-				handle1
-			}
+			BezierHandles::Quadratic { handle } => handle,
+			BezierHandles::Cubic { handle1, .. } => handle1,
 		}
 	}
 
 	pub fn get_handle2(&self) -> Option<DVec2> {
 		match self.handles {
-			BezierHandles::Quadratic { handle: _ } => {
-				None
-			}
-			BezierHandles::Cubic { handle1: _, handle2 } => {
-				Some(handle2)
-			}
+			BezierHandles::Quadratic { .. } => None,
+			BezierHandles::Cubic { handle2, .. } => Some(handle2),
 		}
 	}
 
-
 	pub fn get_points(&self) -> [Option<DVec2>; 4] {
 		match self.handles {
 			BezierHandles::Quadratic { handle } => [Some(self.start), Some(handle), Some(self.end), None],
 			BezierHandles::Cubic { handle1, handle2 } => [Some(self.start), Some(handle1), Some(handle2), Some(self.end)],
 		}
 	}
 
-	/// Return the length of the bezier curve
-	pub fn length() -> i32 {
-		0
-	}
-}
+	///  Calculate the point on the curve based on the t-value provided
+	///  basis code based off of pseudocode found here: https://pomax.github.io/bezierinfo/#explanation
+	pub fn get_basis(&self, t: f64) -> DVec2 {
+		let t2 = t * t;
+		let mt = 1.0 - t;
+		let mt2 = mt * mt;
```

Comment:
Perhaps try and expand some of these abbreviations in line with graphite style guidelines.

Reply:
(for reference: https://graphite.rs/contribute/#contributing-guide although I still need to expand that with more details)

Reply:
I'm not super sure what's the best way to expand these since they're mainly used to simplify the line of code that does the mathematical computation. Would it be better to use names such as:
- `t_squared` instead of `t2`
- `one_minus_t` instead of `mt`
- `one_minus_t_squared` instead of `mt2`
- etc?

Reply:
Those would be great names, yes :)

Reply:
I've updated the names - is this better?

Reply:
Sure, looks good!